README.md

```diff
@@ -14,107 +14,87 @@ Assumptions:
 
 ### Install Prerequisites and Code
 
-1. If you're not already running Node.js, download and run the installer for your operating system. We recommend v4.x: [https://nodejs.org/en/](https://nodejs.org/en/ "Node.js installer").
+1. If you're not already running Node.js, download and run the installer for your operating system. We recommend v10.x: [https://nodejs.org/en/](https://nodejs.org/en/ "Node.js installer").
 
-2. Use *npm* (Node's package manager, which is part of the Node.js install) to install the latest version of gulp. This is necessary for runing the babel version of the `gulpfile`):
-
-    ```bash
-        $ npm install gulp && npm install gulp -g
-    ```
-
-3. From the command line, clone the DATA Act web app repository from GitHub to your local machine:
+2. From the command line, clone the DATA Act web app repository from GitHub to your local machine:
 
         $ git clone https://github.com/fedspendingtransparency/data-act-broker-web-app.git
 
-4. Switch to the alpha release version of the code. This is the latest stable release.
+3. Switch to the alpha release version of the code. This is the latest stable release.
 
         $ git checkout v0.1.0-alpha
 
     **Note:** If you'd rather use the latest, work-in-progress features of the DATA Act broker, replace the above command with `git checkout staging`.
 
-5. Change to the `data-act-broker-web-app` directory that was created when you cloned the DATA Act web repository.
+4. Change to the `data-act-broker-web-app` directory that was created when you cloned the DATA Act web repository.
 
-6. Install the web application's package dependencies:
+5. Install the web application's package dependencies:
 
         $ npm install
 
 
 ### Create Configurations
 
-The `data-act-broker-web-app` folder provides three sample `GlobalConstants` files:
-
- * `sampleGlobalConstants_dev.js`
- * `sampleGlobalConstants_local.js`
- * `sampleGlobalConstants_prod.js`.
+The `data-act-broker-web-app` folder provides a configuration object in the `GlobalConstants.js` file. By default, it is configured to point at `http://localhost:9999/v1/`. To modify your configuration, you may overwrite this file directly or use an environment specific file with the following naming convention instead: `GlobalConstants_<local/dev/prod>.js`.
```

Comment:
> By default, it is configured to point at `http://localhost:9999/v1/`.

It would be more clear to specify that this is for the `API`

Reply:
👌